### PR TITLE
Rule update: "aria-hidden with focusable content"

### PR DESCRIPTION
**Pull request by annethyme**
"focusable" --> "Participate in sequential focus navigation"

Closes issues: 
- https://github.com/auto-wcag/auto-wcag/issues/374 (Should "Aria-hidden with focusable content" require that `aria-hidden` elements not participate in sequential focus navigation order rather than be not focusable?) 
- https://github.com/auto-wcag/auto-wcag/issues/314 (aria-hidden with focusable content: Incorrect test case)

"contain" --> "have descendants in the flat tree"

Closes issue: 
- https://github.com/auto-wcag/auto-wcag/issues/315 (aria-hidden with focusable content: Expectation is ambiguous)

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

File: _rules/SC4-1-2-aria-hidden-focus.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The rule applies to any element with an `aria-hidden="true"` attribute.
 
 ### Expectation
 
-None of the target elements are [focusable](#focusable), nor do they contain a [focusable element](#focusable).
+None of the target elements participate in [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sec-sequential-focus-navigation), nor do they have [descendants](https://www.w3.org/TR/dom41/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that participate in [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sec-sequential-focus-navigation)
```
Comment (kasperisager) [inline _rules/SC4-1-2-aria-hidden-focus.md]: Missing stop at the end.

**Comment by annethyme** [comment]
@audreymaniez, @jkodu, @kasperisager, @ShadowBB:

After some internal discussions, I have added a test case with tabindex="-2". 

I hope you can still approve.

**Comment by DagfinnRomen** [comment]
What is the difference between this (#439) and #449?

**Comment by annethyme** [comment]
@DagfinnRomen, #439 and #449 solve two different issues for the same rule. They should not be seen as competing suggestions. In an ideal world, it would have been great if they were handled together, but since there are no conflicts between the two, I think it works fine to review and merge them separately.

**Comment by annethyme** [comment]
@WilcoFiers, I have changed "participate in [sequential focus navigation]" to "are part of [sequential focus navigation]" in hope to make it a bit easier to read.

**Pull request merged**